lore/settings.py

```diff
@@ -135,12 +135,20 @@ def get_var(name, default):
 # Uses DATABASE_URL to configure with sqlite default:
 # For URL structure:
 # https://github.com/kennethreitz/dj-database-url
-DATABASES = {
-    'default': dj_database_url.parse(
-        get_var('DATABASE_URL', 'sqlite:///{0}'.format(
-            os.path.join(BASE_DIR, 'db.sqlite3')
-        ))
+DEFAULT_DATABASE_CONFIG = dj_database_url.parse(
+    get_var(
+        'DATABASE_URL',
+        'sqlite:///{0}'.format(os.path.join(BASE_DIR, 'db.sqlite3'))
     )
+)
+# Enable SSL
+if get_var('LORE_USE_DB_SSL', False):
```

Comment:
Do you think it would be slightly safer if we default this to `True` and override this as `False` when running the docker container (or whenever debug is on)?